realm/realm-library/src/objectServer/java/io/realm/ErrorCode.java

```diff
@@ -23,7 +23,7 @@
  */
 public enum ErrorCode {
 
-    // See https://github.com/realm/realm-sync/blob/master/doc/protocol_17.md
+    // See Client::Error in https://github.com/realm/realm-sync/blob/master/src/realm/sync/client.hpp
```

Comment:
From Mortens description it sounds like we need to keep both a reference to `client.hpp` and the protocol docs, since we can get errors reported from both?